algorithmic_efficiency/workloads/cifar/cifar_pytorch/workload.py

```diff
@@ -77,18 +77,19 @@ def _build_dataset(self,
         normalize
     ])
 
-    dataset = CIFAR10(root=data_dir,
-                train=split in ['train','eval_train','validation'],
-                download=True,
-                transform=train_transform_config if "train" in split \
-                                            else eval_transform_config)
+    dataset = CIFAR10(
+        root=data_dir,
+        train=split in ['train', 'eval_train', 'validation'],
+        download=True,
+        transform=train_transform_config
+        if 'train' in split else eval_transform_config)
```

Comment:
nit: can you pull this out into a new line above? I try to avoid multi-line ternary operators